src/canvas-view/canvas-editor/toolbox-items.component.js

```diff
@@ -0,0 +1,31 @@
+(function () {
+  'use strict';
+
+  angular.module('patternfly.canvas')
+    .component('toolboxItems', {
+      templateUrl: 'canvas-view/canvas-editor/toolbox-items.html',
+      bindings: {
+        items: '=',
+        startDragCallback: '<',
+        clickCallback: '<',
+        searchText: '='
+      },
+      controller: function toolboxItemsController () {
+        var ctrl = this;
+
+        ctrl.clickCallbackfmDir = function (item) {
```

Comment:
This is an odd name for the function, what does the 'fm' mean?

Reply:
I have no idea. They were already named these lol

Reply:
That stood for 'from directive'.  I think at some point just 'clickCallback' wasn't descriptive enough.